OpenAI-DotNet-Tests/TestFixture_12_Assistants.cs

```diff
@@ -22,23 +22,19 @@ public async Task Test_01_CreateAssistant()
 
             var assistantFileId = file.Id;
 
-            var request = new CreateAssistantRequest
+            var request = new CreateAssistantRequest("gpt-3.5-turbo-1106")
```

Comment:
Does this need to be turbo in order for the test to pass?

Reply:
well it can be either 3.5-turbo or 4+ model